### ISSUE TITLE
[SUGGEST] Customer Facing Display (Second Monitor)

### ISSUE THREAD

**Issue by wilsontulus** [apinprastya/sultan#68]
Hello there, devs and contributors!

This app looks very promising for many startups and new businesses, as we don't ever have to worry about proprietary software!
However I have one suggestion: There exists a thing, called Customer Facing Display, it's basically a secondary screen that allow customers to look at their purchase logs before a receipt was printed, and in some cases you can put a QR code on the CFD for something like voucher codes, so the customer can just scan the code instead of re-typing the code from the receipt. 
FYI: This also allows receipts to be "opt-out" by default, reducing paper waste!

Anyone interested with this feature is appreciated!
Thanks 😊
(Replies in Indonesian is also appreciated.)

**Comment by wilsontulus** [comment]
I know that it exists in the commit #605a340 but we hope that a new release version will have this feature 😊

**Comment by apinprastya** [comment]
It has the feature but only for Linux for now. No device to check on windows.

**Comment by wilsontulus** [comment]
Oh, alright since i have a debian laptop. and it's compile from source only for now? since the commit is 2021 and latest release is 2020.
I think if I can fix any mistakes i can make a pull request later on 😊

**Comment by wilsontulus** [comment]
Turns out the customer display implementation is not a secondary monitor but rather a more traditional 2-line display thing so-called "customer pole display". After lots of tries and gave up by looking at source code turns out the implementation is not as I expected.. 
The customer display "second monitor" thing looks pretty cool implementation and this also helps if a product involves something like a voucher code, QR code, etc.
maybe a space to improve this project? I can contribute as I learn and understand more about C++ and how this awesome FOSS software works 😅

**Comment by apinprastya** [comment]
Yes, it does not support the multiple display yet. That will need another window to other screen. And I am currently moving to flutter for the front end, so any user can easily implement their UI as they want. It should be easier to get flutter dev rather C++ dev here.
